modules/common/src/main/java/com/opengamma/sdk/common/ServiceInvoker.java

```diff
@@ -43,6 +43,10 @@
  * such as via the try-with-resources statement.
  */
 public final class ServiceInvoker implements AutoCloseable {
+  /**
+   * The User Agent header to attach to all outgoing HTTP requests.
+   */
+  public static final String USER_AGENT = "OG-JAVA-SDK";
```

Comment:
That sounds good. If we can extract the JVM version and OS as the 'other info' that might be useful.